packages/webapp/src/routes/land/add-landowner-organisation.js

```diff
@@ -0,0 +1,55 @@
+import constants from '../../utils/constants.js'
+import {
+  getLegalAgreementDocumentType
+} from '../../utils/helpers.js'
+
+const handlers = {
+  get: async (request, h) => {
```

Comment:
in progress indicator missing  processRegistrationTask(request, {
      taskTitle: 'Legal information',
      title: 'Add legal agreement details'
    }, {
      inProgressUrl: 
    })

Reply:
done